logging/logging.go

```diff
@@ -419,19 +420,34 @@ func (instance *Logging) sendErrorResponse(errorStr, logID string) {
 func (instance *Logging) addServiceIDFilter(journal *sdjournal.Journal,
 	fieldName, serviceID string) (unitName string, err error) {
 	if serviceID == "" {
-		return unitName, nil
+		return unitName, aoserrors.New("serviceID is empty")
 	}
 
 	service, err := instance.serviceProvider.GetService(serviceID)
 	if err != nil {
-		return unitName, nil
+		return unitName, err
 	}
 
-	if err = journal.AddMatch(fieldName + "=" + service.UnitName); err != nil {
-		return unitName, aoserrors.Wrap(err)
+	if fieldName == unitField {
+		if err = journal.AddMatch(fieldName + "=" + service.UnitName); err != nil {
+			return unitName, aoserrors.Wrap(err)
+		}
+	} else if fieldName == serviceField {
+		// with cgroup v2 logs from container do not contains _SYSTEMD_UNIT due to restrictions
+		// that's why id should be extracted from _SYSTEMD_CGROUP
+		// for supporting v1 need to add filter also for serviceName
+		// format: /system.slice/aos_AOS_SERVICE_UUID.service
+		if err = journal.AddMatch(fieldName + "=/system.slice/" + service.UnitName); err != nil {
+			return unitName, aoserrors.Wrap(err)
+		}
+
+		// format: /system.slice/AOS_SERVICE_UUID
+		if err = journal.AddMatch(fieldName + "=/system.slice/" + serviceID); err != nil {
```

Comment:
Why both format is needed?

Reply:
i'll update comment inside of function.
It is necessary because:
cgroup v1 has format: /system.slice/aos_AOS_SERVICE_UUID.service
cgroupt v2 has format: /system.slice/AOS_SERVICE_UUID

so it is required for supporting v1 and v2 versions of cgroups